packages/verification-vue-component/src/style/components/verification/_dropbox.scss

```diff
@@ -52,4 +48,41 @@
             }
         }
     }
+
+    &.dragover {
+        position: absolute;
+        top: 0;
+        height: 100%;
+        width: 100%;
```

Comment:
To stretch a div over the whole area, I would use `top: 0; right: 0; bottom: 0; left: 0;` instead of `width: 100%; height; 100%;`